### PR TITLE
Fix build for release 7.12

### PR DESCRIPTION
**Pull request by kozlovsky**
This PR fixes various build problem for release 7.12

### CODE REVIEW

File: src/tribler/gui/code_executor.py [hunk 1]
```diff
@@ -60,7 +71,7 @@ def run_code(self, code, task_id):
             pass
 
         if self.shell.last_traceback:
-            self.on_crash(self.shell.last_traceback)
+            self.on_crash(self.shell.last_traceback + f'\n\ntask_id: {task_id!r}\ncode:\n{code}\n\n(end of code)')
```
Comment (drew2a) [inline src/tribler/gui/code_executor.py:74]: **nit** maybe this version is more readable:
```suggestion
            self.on_crash(f'{self.shell.last_traceback}\n\ntask_id: {task_id!r}\ncode:\n{code}\n\n(end of code)')
```

**Review by drew2a — APPROVED** [review]

File: build/win/makedist_win.bat [hunk 2]
```diff
@@ -51,7 +51,7 @@ REM packs them in the installer .EXE
 ECHO Install pip dependencies for correct py-installer's work
 python3 -m pip install --upgrade -r build\win\requirements.txt
 
-%PYTHONHOME%\Scripts\pyinstaller.exe tribler.spec
+%PYTHONHOME%\Scripts\pyinstaller.exe tribler.spec --log-level=DEBUG
```
Comment (xoriole) [inline build/win/makedist_win.bat:51]: Lets set DEBUG level on other the builder scripts (MacOS and Linux)

File: requirements-core.txt [hunk 3]
```diff
@@ -14,7 +14,7 @@ networkx==2.6.3
 pony==0.7.14
```
Comment (xoriole) [inline requirements-core.txt:14]: Lets update Pony to support Python 3.10

**Review by xoriole — COMMENTED** [review]
Nice fixes!

File: src/tribler/gui/code_executor.py (hunk 1 above)
Comment (kozlovsky) [reply src/tribler/gui/code_executor.py]: Indeed!

File: build/win/makedist_win.bat (hunk 2 above)
Comment (kozlovsky) [reply build/win/makedist_win.bat]: Done!

File: requirements-core.txt (hunk 3 above)
Comment (kozlovsky) [reply requirements-core.txt]: Done!

**Pull request merged**